### PR TITLE
Use external hdd as log storage

### PR DESCRIPTION
**Pull request by fheslouin**
Pretty much the same idea as #170, however the log_to_hdd.yml tasks are now executed during "custom" tag

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -86,9 +82,16 @@
         and cache_machine.state|default(omit) == "present"
       tags: ['update','idc_import']
 
+    - role: logs
+      when: ansible_local.installed_software.management.managed_by_bsf|bool == True
+
```
Comment (swbsf) [inline main.yml:87]: no tags, not sure this is what you wanted?

File: roles/logs/tasks/logs_to_hdd.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - var-log.mount
+
+- name: Enable mounts
+  systemd:
+    enabled: yes
+    name: "{{ item }}"
+  with_items:
+    - media-hdd.mount
+    - var-log.mount
+  notify:
```
Comment (swbsf) [inline roles/logs/tasks/logs_to_hdd.yml:43]: Same here, for the whole file, no tags?

File: roles/logs/tasks/main.yml [hunk 3]
```diff
@@ -41,14 +48,21 @@
...
     dest: /etc/default/anacron
     regexp: '^ANACRON_RUN_ON_BATTERY_POWER'
     line: 'ANACRON_RUN_ON_BATTERY_POWER=yes'
     state: present
+  tags: ['custom','update']
 
+- include: logs_to_hdd.yml
+  when: ext_hdd
+  tags:
+    - custom
```
Comment (swbsf) [inline roles/logs/tasks/main.yml:68]: Ok, here it is :)

**Review by swbsf — APPROVED** [review]
Seems good to me.

File: roles/logs/files/media-hdd.mount [hunk 4]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Disque dur externe du Koombook
```
Comment (letompouce) [inline roles/logs/files/media-hdd.mount]: Tiens, du français.

File: main.yml (hunk 1 above)
Comment (fheslouin) [reply main.yml:87]: You'll find them within the role

File: roles/logs/tasks/logs_to_hdd.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+- name: Create probably missing log folders on external HDD
+  file:
+    path: "{{ bsf_log_folder }}/log/{{ item.name }}"
+    owner: "{{ item.owner }}"
+    group: "{{ item.group }}"
+    mode: 0775
```
Comment (letompouce) [inline roles/logs/tasks/logs_to_hdd.yml:7]: 0755 for these directories. `/var/log/bsf/` being an exception (which I don't like but that's another topic).

Comment (fheslouin) [reply roles/logs/tasks/logs_to_hdd.yml:7]: What do you recommend then ?

File: roles/logs/files/var-log.mount [hunk 6]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Montage répertoire de log
```
Comment (letompouce) [inline roles/logs/files/var-log.mount]: Should be « Journaling Verzeichnis erstellen » or something similar :)

File: roles/logs/tasks/logs_to_hdd.yml (hunk 5 above)
Comment (letompouce) [reply roles/logs/tasks/logs_to_hdd.yml:7]: Currently writable by `adm` group. Only `telegraf` in this group (even `ideascube` is not, despite being suder :p). I am not fond of adding system users to the `adm` group.

The common practice is to create a dedicated group, see `/etc/group` for a tons of examples.

Comment (letompouce) [reply roles/logs/tasks/logs_to_hdd.yml:7]: Let me put some emphasis about this:

My remark is beyond the scope of this pull request, and pull requests should be as atomic as commits should be. Let's keep it like that in here, and fix the group/ownership later.

Comment (fheslouin) [reply roles/logs/tasks/logs_to_hdd.yml:7]: Ok, let's do that

**Pull request merged**